### PR TITLE
Updated readme to follow WG repos

### PR DESCRIPTION
**Pull request by himorin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +96,18 @@ The mapping of this functionality to traditional flat displays is fairly clear -
 ## Could this be used for VR modes?
 
 In a word, maybe. This functionality is based on the fact that all AR sessions are “immersive,” even on smartphones. If this exploration is successful, it may be worth considering whether “immersive-vr” should also be supported on traditional displays and allow such an overlay to be specified. As part of this VR exploration, we may want to consider the usefulness vs. the complexity for VR implementations and the likely adoption by implementers.
+
+<!-- Links -->
+[CoC]: https://immersive-web.github.io/homepage/code-of-conduct.html
+[webxrwg]: https://w3.org/immersive-web
+[cgproposals]: https://github.com/immersive-web/proposals
+[webxrspec]: https://immersive-web.github.io/webxr/
+[webxrcg]: https://www.w3.org/community/webvr/
```
Comment (dontcallmedom) [inline README.md:105]: should use the updated url https://www.w3.org/community/immersive-web/

File: README.md [hunk 2]
```diff
@@ -50,3 +96,18 @@ The mapping of this functionality to traditional flat displays is fairly clear -
...
+[wgcharter]: https://www.w3.org/2020/05/immersive-Web-wg-charter.html
+[webxrref]: https://immersive-web.github.io/webxr-reference/
+[wghome]: https://w3.org/immersive-web
```
Comment (dontcallmedom) [inline README.md:108]: this is a dup of the ref webxrwg

File: README.md [hunk 3]
```diff
@@ -50,3 +96,18 @@ The mapping of this functionality to traditional flat displays is fairly clear -
...
+[publiclist]: https://lists.w3.org/Archives/Public/public-immersive-web/
```
Comment (dontcallmedom) [inline README.md:109]: should this point to the -wg list given that this a wg item? (maybe it's fine to keep it on the general list, I'm just unsure)

Comment (himorin) [reply README.md:109]: ah, I think it should be. this is a list for CG..

File: README.md (hunk 1 above)
Comment (himorin) [reply README.md:105]: will update.

File: README.md (hunk 2 above)
Comment (himorin) [reply README.md:108]: YES! thank you for catching this.

**Comment by AdaRoseCannon** [comment]
The reference to the spec has been removed, but not replaced as far as i can tell, it would be great to have a link to the compiled spec from the readme.

**Comment by himorin** [comment]
> The reference to the spec has been removed, but not replaced as far as i can tell, it would be great to have a link to the compiled spec from the readme.

Ah, yes. I haven't noticed that I forgot to include that part. Let me update PR(s; should be also for hit-test) to add "Specifications" section.

**Pull request merged**